api.js

```diff
@@ -180,6 +186,25 @@ Api.prototype._prefixTitle = function (file) {
 	return prefix;
 };
 
+Api.prototype._restartTimer = debounce(function () {
```

Comment:
There is no need for this function using `debounce`.

Just do the following in the constructor:

```js
if (this.options.timeout) {
  this._onTimeout = debounce(this._onTimeout, ms(this.options.timeout));
} else {
  this._onTimeout = noop;
}
```

I think that will be easier to understand (and faster) than the current implementation or the `no-clear` branch.